externalexamples/readme.txt

```diff
@@ -1,7 +1,6 @@
-This directory is intended to contain code for projects that use Umple, but are not
+This directory is intended to contain code for projects that use Umple but are not
 part of the Umple project itself.
 
-We invite people to create their own umple examples, with proper test cases included.
-We can then build them automatically. That way we can help ensure that as we update
-the compiler it doesn't break your code, and also your code serves as additional
-test cases for us.
+We invite people to create their own umple exaples, including proper test cases. 

```

Comment:
Please consider the following cases,
- Umple should be started always with an uppercase letter `U`
- There is a typo in your modification  -> `exaples`